scripts/deploy.sh

```
@@ -0,0 +1,179 @@
+        if lsof -ti:$port > /dev/null 2>&1; then
+            kill -9 $(lsof -ti:$port)
+        fi
+    fi
+}
+
+# 앱 시작
+start_app() {
+    local port=$1
+    log "Starting application on port $port"
+
+    cd $DEPLOY_PATH
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)